test/highlight/functions.gleam

```diff
@@ -23,14 +23,14 @@ pub fn replace(
   // ^ type
   //        ^ punctuation.bracket
   string.replace(in: string, each: pattern, with: replacement)
-  // <- variable
+  // <- variable.parameter
```

Comment:
Ah, this is a little bit funny. Here `string` is actually a module name, but it is being highlighted as a parameter because there's a parameter of the same name. While this feels like a mistake, it appears that the Gleam compiler also doesn't like the ambiguity (`string` _could_ be a record). I think I got it to compile once, but adding after adding a type to the file, it no longer compiles.

Anyhow, the TL;DR here is: Let's change the parameter `string` to, say,  `original` so it's not ambiguous :grin:

Reply:
ah good catch, I agree :+1: